project.py

```diff
@@ -0,0 +1,153 @@
+from bs4 import BeautifulSoup
+import requests
+import re
+import pickle
+import os.path
+from pathlib import Path
+import random
+
+def dict_creation_one(script, things = dict()):
+    '''Create the base dictionary for ONE word; only to be called if there is no dicts1.p already'''
+    entiredict =  things
+    everything = script
+    newstring = ''
+    for x in script: #get rid of symbols that I don't want in my speech
+        if x not in ('[', ']', '@', '#', '$', '%', '^', '&', '*', '(', ')', ':', '"', '\\', '—', '|', '-'):
+            newstring += (x)
+    newstring = newstring.lower()
+    listowords = newstring.split()
+    index = 0
+    for y in listowords:    #run through all of the words and create a dictionary with them
```

Comment:
If i read this correctly, you are trying to make a dictionary that is `word : [list of words that come after it`? A less general comment would be helpful.